core/embed/projects/prodtest/README.md

```
@@ -1712,22 +1712,23 @@ OK 2025 07 03 14 23 00 4
+`OK <min_temp_c> <max_temp_c> <battery_errors> <battery_cycles> <tropic_alarms>`
 
 - `min_temp_c`: Minimum temperature in millidegrees Celsius (°C × 1000)
 - `max_temp_c`: Maximum temperature in millidegrees Celsius (°C × 1000)
 - `battery_errors`: Battery error flags as hexadecimal
 - `battery_cycles`: Battery cycles in millicycles (cycles × 1000)
+- `tropic_alarms`: The cumulative number of alarms that the Tropic chip raised
 
 If telemetry data is not available (not yet initialized), the command returns an error.
 
 Example:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)